mlir/test/Target/LLVMIR/llvmir-debug.mlir

```diff
@@ -448,7 +448,7 @@ llvm.func @func_debug_directives() {
 #di_compile_unit = #llvm.di_compile_unit<id = distinct[1]<>, sourceLanguage = DW_LANG_C, file = #di_file, isOptimized = false, emissionKind = None>
 
 // Recursive type itself.
-#di_struct_self = #llvm.di_composite_type<tag = DW_TAG_null, recId = distinct[0]<>>
+#di_struct_self = #llvm.di_composite_type<recId = distinct[0]<>, isRecSelf = false>
```

Comment:
Did you mean to set it to `true` instead? I guess we didn't really check in the rec-self case, but might be good to set it so it's not confusing.

Reply:
That was a classic search replace thingy just saw there are more such instances. Will fix them.